### PR TITLE
TASK: Improve wording of include/exclude concepts where possible

### PR DESCRIPTION
**Pull request by albe**
Matching change with https://github.com/neos/flow-development-collection/pull/2024

### CODE REVIEW

File: Neos.ContentRepository/Classes/Domain/NodeType/NodeTypeConstraints.php [hunk 1]
```diff
@@ -78,14 +78,14 @@ public function getExplicitlyDisallowedNodeTypeNames(): array
...
-        // if $nodeTypeName is explicitely blacklisted, it is DENIED.
+        // if $nodeTypeName is explicitely excluded, it is DENIED.
         foreach ($this->explicitlyDisallowedNodeTypeNames as $disallowed) {
             if ((string)$nodeTypeName === (string)$disallowed) {
                 return false;
             }
         }
 
-        // if $nodeTypeName is explicitely whitelisted, it is ALLOWED.
+        // if $nodeTypeName is explicitely allowed, it is ALLOWED.
```
Comment (albe) [inline Neos.ContentRepository/Classes/Domain/NodeType/NodeTypeConstraints.php:88]: That's what happens when you get rid of loaded terminology.

File: Neos.ContentRepository/Tests/Behavior/Features/Security/EditNodePrivilegeBasedOnDimension.feature [hunk 2]
```diff
@@ -11,7 +11,7 @@ Feature: Privilege to restrict editing of nodes for a single dimension only
           'Neos.ContentRepository:EditServiceNodes':
             matcher: 'isDescendantNodeOf("/sites/content-repository/service")'
 
-          # EditEverything is needed, to switch to a "WHITELIST MODE" - i.e. where everything must be allowed explicitely.
+          # EditEverything is needed, to switch to a "ALLOWLIST MODE" - i.e. where everything must be allowed explicitely.
```
Comment (albe) [inline Neos.ContentRepository/Tests/Behavior/Features/Security/EditNodePrivilegeBasedOnDimension.feature:14]: Note: "Allowlist" was argued against in the Flow PR, since it is not a correct english term. Open for suggestions.

Comment (theilm) [reply Neos.ContentRepository/Tests/Behavior/Features/Security/EditNodePrivilegeBasedOnDimension.feature:14]: What about "EXPLICIT ALLOW MODE"? (corresponds to the second part of the comment as in the first change.)

Comment (albe) [reply Neos.ContentRepository/Tests/Behavior/Features/Security/EditNodePrivilegeBasedOnDimension.feature:14]: Guess that would work, and then we can probably even scrap off the explanatory second part after the dash (see also my comment above). Thanks!
```suggestion
          # EditEverything is needed, to switch to an "EXPLICIT ALLOW MODE"
```

File: Neos.ContentRepository/Classes/Domain/NodeType/NodeTypeConstraints.php (hunk 1 above)
Comment (bwaidelich) [reply Neos.ContentRepository/Classes/Domain/NodeType/NodeTypeConstraints.php:88]: Maybe we can make this a little bit clearer like:
```// if $nodeTypeName is explicitely allowed, it is ALLOWED. IT. IS. ALLLOOOOWED!!!1!1!!```

;)

File: Neos.Media.Browser/Resources/Public/JavaScript/Main.js [hunk 3]
```diff
@@ -236,27 +236,27 @@ if (undefined === 'production') {
...
+	// This works if the window reference is available
+	if(typeof window === "object")
+		g = window;
+}
+
+// g can still be undefined, but nothing to do about it...
+// We return undefined, instead of nothing here, so it's
+// easier to handle this case. if(!global) { ...}
+
+module.exports = g;
```
Comment (bwaidelich) [inline Neos.Media.Browser/Resources/Public/JavaScript/Main.js:259]: Is this change related?

File: Neos.ContentRepository/Tests/Behavior/Features/Security/EditNodePrivilegeBasedOnDimension.feature (hunk 2 above)
Comment (bwaidelich) [reply Neos.ContentRepository/Tests/Behavior/Features/Security/EditNodePrivilegeBasedOnDimension.feature:14]: Makes sense IMO

File: Neos.ContentRepository/Classes/Domain/NodeType/NodeTypeConstraints.php (hunk 1 above)
Comment (bwaidelich) [reply Neos.ContentRepository/Classes/Domain/NodeType/NodeTypeConstraints.php:88]:
```suggestion
        // if $nodeTypeName is explicitely ALLOWED.
```

File: Neos.Media.Browser/Resources/Public/JavaScript/Main.js (hunk 3 above)
Comment (albe) [reply Neos.Media.Browser/Resources/Public/JavaScript/Main.js:259]: Huh? No, definitely not. Where does this come from?

Comment (kdambekalns) [reply Neos.Media.Browser/Resources/Public/JavaScript/Main.js:259]: Trailing CR are being removed:
Screenshot 2020-08-11 at 18 57 14
Thats a good thing.
